src/components/Shell/ShellAppBar.tsx

```diff
@@ -89,8 +108,12 @@ export const ShellAppBar = ({
           </IconButton>
         </Tooltip>
         {doShowPeers ? (
-          <Tooltip title="Number of peers in the room">
-            <StepIcon icon={numberOfPeers} sx={{ ml: 2 }} />
+          <Tooltip title="Clic to show peer list">
```

Comment:
"Click" is misspelled:

```suggestion
          <Tooltip title="Click to show peer list">
```